packages/xod-fs/src/messages.js

```diff
@@ -0,0 +1,10 @@
+import { PROJECT_LOADED_WITH_INVALID_PATCH_PATHS } from './errorCodes';
+
+export default {
+  [PROJECT_LOADED_WITH_INVALID_PATCH_PATHS]: ({ patchPaths }) => ({
+    title: 'Can not load some patches',
+    note: `Some patches have an invalid patch paths: ${patchPaths.join(', ')}`,
```

Comment:
I’d limit this array with an ellipsis if more than, say, three items. Otherwise, UI will explode